### PR TITLE
Supplement on multipleviews scenes data reading

### PR DESCRIPTION
**Pull request by Hongyuan-Tao**
I added a new function called readMultipleViewinfos to held reading the multipleviews scenes data.

### CODE REVIEW

File: multipleviewprogress.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+python downsample_point.py ./colmap/dense/fused.ply ./data/$workdir/points3D_multipleview.ply
+
+git clone https://github.com/Fyusion/LLFF.git
+pip install scikit-image
+python LLFF/imgs2poses.py scenedir ./colmap/
+
+cp ./colmap/poses_bounds.npy ./data/$workdir/poses_bounds_multipleview.npy
+
+rm -r ./colmap
```
Comment (guanjunwu) [inline multipleviewprogress.sh:25]: just change the folders name, such as `colmap_tmp` or `tmp`.

File: multipleviewprogress.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+workdir=$1
+export CUDA_VISIBLE_DEVICES=0
```
Comment (guanjunwu) [inline multipleviewprogress.sh:2]: maybe delete this line.

**Review by guanjunwu — COMMENTED** [review]
@taoranyi just take a look!
And the ` ./data/$workdir/poses_bounds_multipleview.npy` maybe can be laoded by `dynerf`'s dataloader? Do we need another independent dataloader to load the dataset?

File: README.md [hunk 3]
```diff
@@ -88,6 +88,29 @@ Meanwhile, [Plenoptic Dataset](https://github.com/facebookresearch/Neural_3D_Vid
 |     ├── ...
 ```
 
+**For multipleviews scenes:**
+The dataset privided in [Ballett5](https://www.swisstransfer.com/d/b3f1ebc8-def6-428f-924a-9a132a86a2f9) is used.If you want to train your own dataset of multipleviews scenes,you can orginize your dataset like Ballett5 as follows:
```
[the remark this answers is not in the document]
Comment (guanjunwu) [reply README.md:92]: delete the dataset and link, it is not for public use. then using `git rebase` rebase all the commit in to 1 commit.

File: README.md [hunk 4]
```diff
@@ -88,6 +88,41 @@ Meanwhile, [Plenoptic Dataset](https://github.com/facebookresearch/Neural_3D_Vid
...
+│   | cam02
+│     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | ...
+│   | sparse_
+│     ├── cameras.bin
+│     ├── images.bin
+│   | points3D_multipleview.ply
+│   | poses_bounds_multipleview.npy
```
Comment (guanjunwu) [inline README.md:122]: And the ` ./data/$workdir/poses_bounds_multipleview.npy` maybe can be laoded by `dynerf`'s dataloader? Do we need another independent dataloader to load the dataset?

File: README.md [hunk 5]
```diff
@@ -88,6 +88,41 @@ Meanwhile, [Plenoptic Dataset](https://github.com/facebookresearch/Neural_3D_Vid
...
+|     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | cam02
+│     ├── frame_00001.jpg
+│     ├── frame_00002.jpg
+│     ├── ...
+│   | ...
+```
+After that,you can use the  `multipleviewprogress.sh` I provided to generate related data of poses and pointcloud.You need to
```
Comment (taoranyi) [inline README.md:105]: Do not use "I", use we.

File: README.md [hunk 6]
```diff
@@ -88,6 +88,41 @@ Meanwhile, [Plenoptic Dataset](https://github.com/facebookresearch/Neural_3D_Vid
...
+│   | ...
+│   | sparse_
+│     ├── cameras.bin
+│     ├── images.bin
+│   | points3D_multipleview.ply
+│   | poses_bounds_multipleview.npy
+```
+
+
 ## Training
```
Comment (taoranyi) [inline README.md:126]: In "Training" section, please add how to run on this dataset.

**Review by taoranyi — COMMENTED** [review]
Edit readme

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hongyuan-Tao) [reply README.md:92]: ok，let me take a look

[the remark this answers is not in the document]
Comment (Sabisisi) [reply README.md:92]: hi guys, pls take the link to the dataset down, wasn't meant to be shared publicly

**Pull request merged**